versioning/scripts/dockerfiles/main.go

```diff
@@ -173,30 +180,39 @@ func check(e error) {
 func main() {
 	templateDirPtr := flag.String("template_dir", "templates", "Path to directory containing Dockerfile.template and any other files to copy over")
 	verifyPtr := flag.Bool("verify_only", false, "Verify dockerfiles")
+	var templatePath string
+	var version = "0.01"
+
+	log.Printf("dockerfiles verson: %s", version)
+
 	flag.Parse()
 
 	var spec versions.Spec
 	spec = versions.LoadVersions("versions.yaml")
 
-	templatePath := filepath.Join(*templateDirPtr, "Dockerfile.template")
-	templateData, err := ioutil.ReadFile(templatePath)
-	templateString := string(templateData)
-	check(err)
+	for _, version := range spec.Versions {
+		if version.TemplateDir != "" {
```

Comment:
I think we're gonna need an extra variable `templateDir` which would evaluate to either
- `filepath.Join(*templateDirPtr, version.TemplateDir)`, or
- `*templateDirPtr`

And `templatePath` should simply be `filepath.Join(templateDir, "Dockerfile.template")`.